src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -69,9 +69,9 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
 
         // Keywords match phone, email and address, but does not match name
```

Comment:
Should this comment be updated to reflect new changes in case it causes confusion? I noticed a similar issue in other places (mainly in regards to phone and address)

Reply:
I was largely optimising to ensure I could push the add feature out ASAP so its easier to work on the rest of the feature, is this something you feel we need to edit as of now? or we can leave this to a later time.